### PR TITLE
Create governance-art-collection.md

### PR DESCRIPTION
**Pull request by OfficialCryptos**

### CODE REVIEW

File: governance-art-collection.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+| Title               | Governance Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
```
Comment (OfficialCryptos) [inline governance-art-collection.md:3]: Featured text?
Something like:
"Decred got the art of governance down"

Comment (jpsantanna) [reply governance-art-collection.md:3]: I like the featured text idea by Phoenix, but i'll leave two more:

1- Witness the evolution of decentralization as Decred meticulously crafts its governance DNA

2- Unravel the DNA of decentralization with Decred's pioneering governance model

**Comment by xaur** [comment]
This PR was merged by accident before it was finished. Replaced by #25, since this PR cannot be reopened after it was merged.

File: governance-art-collection.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+| Author              | OfficialCryptos |
+| Featured text       |Unravel the DNA of decentralization with Decred's pioneering governance model|
+
+Thesis statement for art collection - What the article is about, and what is it aiming to prove
+
+# Article Title
+
+## Art Piece 1 Breaking the chains: Decred liberates itself from centralization by empowering coin holders to participate in steering the direction of its chain
+
+## Art Piece 2 + Decred has a unique governance system in place that makes this a DAO governed like clockwork
```
[the remark this answers is not in the document]
Comment (PhoenixGreen) [reply governance-art-collection.md:11]: Great images. The interconnection between coin holders and governance, brilliant 👍

[the remark this answers is not in the document]
Comment (OfficialCryptos) [reply governance-art-collection.md:11]: Thanks! 👍

**Pull request merged**